.mailmap

```diff
@@ -51,21 +59,31 @@ Samaiyah I. Farid <38890767+samaiyahfarid@users.noreply.github.com>
 Sean Carroll <seanwilliamcarroll@gmail.com>
 Sean Chambers <schambers80@gmail.com>
 Siddharth <think.siddharth@gmail.com>
+Sjbrownian <84342922+Sjbrownian@users.noreply.github.com>
+Steve Richardson <arichar6@gmail.com>
 Stuart Mumford <stuart@cadair.com>
 Thomas Fan <thomasjpfan@gmail.com>
 Thomas Ulrich <tmulrich1@gmail.com>
 Thomas Varnish <5612615+tvarnish@users.noreply.github.com>
 Tiger Du <56400881+Tiger-Du@users.noreply.github.com>
+Tomás Stinson <14tstinson@gmail.com>
 Volken <the.shadow.anonyme@gmail.com>
+W. Cody Skinner <cody@codyskinner.net>
 Yi-Min Huang <yopology@yahoo.com>
+bryancfoo <68606269+bryancfoo@users.noreply.github.com>
 cclauss <cclauss@bluewin.ch>
+dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
 flaixman <biel_nadal@hotmail.com>
 goodab <goodab@uw.edu>
+haman80 <70356097+haman80@users.noreply.github.com>
 hzxusx <hzxusx@gmail.com>
 jams2 <me@joshuamunn.com>
 lgoenner <lgoenner@users.noreply.github.com>
 misupova <marija.isupova@gmail.com>
 nrb1324 <nrb1324@hotmail.com>
+pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>
```

Comment:
```suggestion
flaixman <biel_nadal@hotmail.com>
goodab <goodab@uw.edu>
haman80 <70356097+haman80@users.noreply.github.com>
hzxusx <hzxusx@gmail.com>
jams2 <me@joshuamunn.com>
lgoenner <lgoenner@users.noreply.github.com>
misupova <marija.isupova@gmail.com>
nrb1324 <nrb1324@hotmail.com>
```
[optional] Not a big deal, but we could remove dependabot and pre-commit-ci from the mailmap...though I suppose they both have a right to be included in author lists, perhaps?  🤔